src/channel/mpmc.rs

```diff
@@ -742,27 +758,42 @@ mod if_alloc {
         mod if_std {
             use super::*;
 
+            use crate::buffer::GrowingHeapBuf;
+
             /// A [`GenericSender`] implementation backed by [`parking_lot`].
-            pub type Sender<T> = GenericSender<parking_lot::RawMutex, T>;
+            ///
+            /// Uses a `GrowingHeapBuf` whose capacity grows dynamically up to
+            /// the given limit. Refer to [`GrowingHeapBuf`] for more information.
+            ///
+            /// [`GrowingHeapBuf`]: ../../buffer/struct.GrowingHeapBuf.html
+            pub type Sender<T> =
+                GenericSender<parking_lot::RawMutex, T, GrowingHeapBuf<T>>;
             /// A [`GenericReceiver`] implementation backed by [`parking_lot`].
-            pub type Receiver<T> = GenericReceiver<parking_lot::RawMutex, T>;
+            ///
+            /// Uses a `GrowingHeapBuf` whose capacity grows dynamically up to
+            /// the given limit. Refer to [`GrowingHeapBuf`] for more information.
+            ///
+            /// [`GrowingHeapBuf`]: ../../buffer/struct.GrowingHeapBuf.html
+            pub type Receiver<T> =
+                GenericReceiver<parking_lot::RawMutex, T, GrowingHeapBuf<T>>;
 
-            /// Creates a new channel.
+            /// Creates a new growing channel.
```

Comment:
```suggestion
            /// Creates a new channel with the given buffering capacity
```

I think think mentioning "growing" below us good enough. If we mention it directly here, people might assume it is some kind of special channel.